### ISSUE TITLE
does the parameter "registry-servers" support remote consul servers now?

### ISSUE THREAD

**Issue by tangweichun** [signal18/replication-manager#249]
does the parameter "registry-servers" support remote consul servers now?
for example,i have a consul cluster:192.168.56.10,192.168.56.11,192.168.56.12,
can i set registry-servers='192.168.56.10,192.168.56.11,192.168.56.12'?

**Comment by scrambledb** [comment]
Hello,

Did you already tried it ? the code looks to pass in consul client option all the hosts separated per ","

Thanks 
/svar  

       var opt registry.Options
	if cluster.Conf.RegistryConsul == false || cluster.IsActive() == false {
		return nil
	}
	opt.Addrs = strings.Split(cluster.Conf.RegistryHosts, ",")

**Comment by tangweichun** [comment]
Yeah,I see the code too!I have tried it,but it seems ignored my configuration, only "127.0.0.1" can work.

mrm config:
registry-consul = true
registry-servers = "172.17.5.201,172.17.5.101,172.17.11.242"

mrm error log:
INFO[2018-09-12T11:20:14+08:00] Register consul master ID write_mysql56 with host 172.17.5.101:5306  cluster=mysql56
ERRO[2018-09-12T11:20:14+08:00] Unexpected register error: Put http://127.0.0.1:8500/v1/agent/service/register: dial tcp 127.0.0.1:8500: getsockopt: connection refused  cluster=mysql56
ERRO[2018-09-12T11:20:14+08:00] Unexpected consul deregister error for server 172.17.10.62:5306: Put http://127.0.0.1:8500/v1/agent/service/deregister/12797976265833579341: dial tcp 127.0.0.1:8500: getsockopt: connection refused  cluster=mysql56
INFO[2018-09-12T11:20:14+08:00] Register consul read service  12797976265833579341 172.17.10.62:5306  cluster=mysql56
ERRO[2018-09-12T11:20:14+08:00] Unexpected consul register error for server 172.17.10.62:5306: Put http://127.0.0.1:8500/v1/agent/service/register: dial tcp 127.0.0.1:8500: getsockopt: connection refused  cluster=mysql56
ERRO[2018-09-12T11:20:14+08:00] Unexpected consul deregister error for server 172.17.5.101:5306: Put http://127.0.0.1:8500/v1/agent/service/deregister/4885623366926929227: dial tcp 127.0.0.1:8500: getsockopt: connection refused  cluster=mysql56

**Comment by scrambledb** [comment]
Yes,

Indeed changing this would mean embedding consul agent. 
We welcome contribution or sponsoring.

But anyway i'll keep this feature in mind for the roadmap.
